### PR TITLE
added harvest and debug to statuspage base

### PR DESCRIPTION
**Pull request by benroeder**

### CODE REVIEW

**Comment by marcopaz** [comment]
Thank you for the PR. Please let's try to keep all the service icons at fixed size 100x100 and using as few kb as possible, ideally each icon should be smaller than 10kb. You could use [convert](http://www.imagemagick.org/script/convert.php) and run  `convert -format png -resize 100 -extent 100x100 -gravity center -background transparent -quality 70 <source-image> <dest-image>` on the images, it should be enough.

**Comment by benroeder** [comment]
This should be better

File: isserviceup/services/models/statuspage.py [hunk 1]
```diff
@@ -10,17 +10,21 @@ def get_status(self):
...
+            elif status == 'status-major':
+                return Status.major
+            elif status == 'status-minor':
+                return Status.minor
+            elif status == 'status-maintenance':
+                return Status.maintenance
+            else:
+                raise Exception('unexpected status')
+        except AttributeError as e:
+            print(b)
```
Comment (marcopaz) [inline isserviceup/services/models/statuspage.py:29]: why do you need this? in production it's better to use some system like sentry to intercept the exceptions and the local variables

Comment (benroeder) [reply isserviceup/services/models/statuspage.py:29]: Agree, but when developing on a laptop, it helps

Comment (marcopaz) [reply isserviceup/services/models/statuspage.py:29]: Fine :)

**Pull request merged**